utils/srg_diff.py

```diff
@@ -94,6 +94,21 @@ def _create_template(root_path: str) -> jinja2.Template:
     return template
 
 
+def get_requirements_with_no_cces(sheet: Worksheet, end_row: int) -> list:
+    result = list()
+    for i in range(2, end_row):
+        requirement = sheet[f'F{i}'].value
+        if requirement is None or requirement.strip() == "":
+            continue
+        cce = sheet[f'D{i}'].value
+        if cce is not None and cce.startswith('CCE-') and requirement.strip() != "":
+            continue
+        status = sheet[f'I{i}'].value
+        if status is not None and status.strip() == 'Applicable - Configurable':
+            result.append(requirement.strip())
+    return result
+
+
```

Comment:
You add the executable permissions but you need to add also a shebang on the first line of this file otherwise the shell doesn't know that it's a python script and when you execute eg. `utils/srg_diff.py --help` it runs it as bash script and gets stuck or freezes. Please add a shebang.